src/panel/poi/blocks/StarsSummary.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { capitalizeFirst } from 'src/libs/string';
+import { useI18n } from 'src/hooks';
+
+const StarsSummary = ({ block }) => {
+  const { _, _n } = useI18n();
+
+  if (!(block?.ratings?.[0]?.has_stars === 'yes')) {
+    return null;
+  }
+
+  if (block.ratings[0].nb_stars && block.ratings[0].nb_stars > 0) {
+    return (
+      <span>
+        {' · ' + capitalizeFirst(_n('%d star', '%d stars', block.ratings[0].nb_stars, 'poi'))}
+      </span>
+    );
+  }
+
+  return <span>{' · ' + capitalizeFirst(_('Starred', 'poi'))}</span>;
```

Comment:
I'm not a huge fan of how this block is prefixed with a middle dot, it makes the block dependent of the context in which it is displayed. I'd say that like in `StarsBlock` we'd rather pass the subclass as a parameter and build the whole line here.

Reply:
For the dot I realized after pushing that it should be displayed with css, fixing it

Reply:
edit: ah no, there's another mechanism for displaying the dot in the poi list, I'll use this one instead